test/dlc_tests/sanity/test_pre_release.py

```diff
@@ -180,6 +182,34 @@ def test_framework_and_cuda_version_gpu(gpu, ec2_connection):
     assert cuda_version in cuda_output.stdout.replace(".", "")
 
 
+@pytest.mark.model("N/A")
+@pytest.mark.parametrize("ec2_instance_type", ["c5.4xlarge"], indirect=True)
+def test_dependency_check_cpu(cpu, ec2_connection):
+    container_name = "dep_check_cpu"
+    report_addon = _get_container_name('depcheck-report', cpu)
```

Comment:
the image is tagged with datetime and pr number in PR context, so it will show up along with the file.

i.e. depcheck-report-pr-mxnet-inference-1-6-0-gpu-py36-cu101-ubuntu16-04-pr-555-2020-09-09-20-08-44.html